src/ui/camera.js

```diff
@@ -294,7 +294,7 @@ class Camera extends Evented {
      */
     rotateTo(bearing: number, options?: AnimationOptions, eventData?: Object) {
         return this.easeTo(extend({
-            bearing: bearing
+            bearing
```

Comment:
There's a few places like this and line 210 that could be moved onto one line with this change. I don't know if ESLint can be configured to do that automatically.